molecule/test/functional/conftest.py

```diff
@@ -79,7 +79,7 @@ def skip_test(request, driver_name):
         'gce': supports_gce,
         'hetznercloud': lambda: min_ansible("2.8") and supports_hetznercloud(),
         'linode': lambda: min_ansible("2.8") and supports_linode(),
-        'podman': lambda: min_ansible("2.9"),
+        'podman': lambda: min_ansible("2.8.6"),
```

Comment:
Wow, you managed to get it merged into 2.8.6?

Reply:
yeah, it's there: https://github.com/ansible/ansible/pull/62915